Gemfile

```diff
@@ -4,6 +4,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'json', '2.7.5' if defined?(JRUBY_VERSION) # temporary
```

Comment:
I believe the json fix went out with `2.8.1` as mentioned [here](https://github.com/ruby/json/issues/694#issuecomment-2460393920)

Reply:
Yep, will remove.